utils/dbz_event_to_fhir_config.json

```diff
@@ -109,6 +109,8 @@
       "enabled": "true",
       "title": "Patient",
       "parentTable": "person",
+      "parentForeignKey": "person_id",
+      "childPrimaryKey":"patient_id",
```

Comment:
Do we need childPrimaryKey?

Reply:
Yes, take a look at how it is being used in `FhirConverter.process()`; for example, we match `patient_id` of table `patient` with `person_id` of table `person`.

Reply:
Thanks, makes sense!